examples/llm/benchmarks/README.md

```diff
@@ -72,7 +81,7 @@ Collect the performance numbers as shown on the [Collecting Performance Numbers]
 
 ## Disaggregated Multi Node Benchmarking
 
-*Two H100 80GB x8 nodes are required for this setup.*
+Two H100 80GB x8 nodes are required for this setup.
 
 > [!Note]
 > Nodes used for benchmarking were part of a cluster connected via InfiniBand
```

Comment:
I think we can remove this given the note above - @kthui what do you think?

Reply:
Yes, I think the new note on the prerequisites section should be sufficient.